Tests/SwiftParserTest/ParserTests.swift

```diff
@@ -6,9 +6,13 @@ import SwiftParser
 public class ParserTests: XCTestCase {
   /// Run a single parse test.
   func runParseTest(fileURL: URL, checkDiagnostics: Bool) throws {
-    let fileContents = try String(contentsOf: fileURL)
-    let parsed = try Parser.parse(source: fileContents)
-    AssertStringsEqualWithDiff("\(parsed)", fileContents)
+    let fileContents = try Data(contentsOf: fileURL)
+    let parsed = try fileContents.withUnsafeBytes({ buffer in
+      try Parser.parse(source: buffer.bindMemory(to: UInt8.self))
+    })
+    // FIXME: This should compare binaries for handling invalid UTF-8 sequences.
```

Comment:
Use the new "syntaxTextBytes" here!

Reply:
Introduced `AssertDataEqualWithDiff`. How does this look?

Reply:
LGTM.  There are a handful of disabled tests in https://github.com/apple/swift-syntax/pull/764 that could be enabled by your change here.

Reply:
I will rebase this PR and try enabling them after #764 lands